sync-template.sh

```diff
@@ -0,0 +1,424 @@
+#!/bin/bash
+# SPDX-License-Identifier: MIT
+
+set -euo pipefail
+
+ME=$(basename $0)
+HERE=${PWD}
+WORKDIR="${HOME}/.cache/${ME%.*}"
+STDERR=/tmp/${ME%.*}.stderr
```

Comment:
Files in `/tmp` need to be created with mktemp for security reasons, see for example: https://en.wikipedia.org/wiki/Symlink_race